firebase-functions/functions/datacite.js

```diff
@@ -184,3 +185,567 @@ exports.getDoiStatus = functions.https.onCall(async (data) => {
   }
 
 });
+
+
+exports.createDraftDoiPR78 = functions.https.onCall(async (data) => {
+
+  const { record, region } = data;
+
+  let prefix;
+  let authHash
+
+  try {
+    prefix = (await admin.database().ref('admin').child(region).child("dataciteCredentials").child("prefix").once("value")).val();
```

Comment:
why not use getCredentialsStored here so you don't have to update db paths in multiple places?